parsl/app/bash.py

```diff
@@ -97,9 +97,7 @@ def open_std_fd(fdname):
 
     missing = []
     for outputfile in kwargs.get('outputs', []):
-        fpath = outputfile
-        if type(outputfile) != str:
-            fpath = outputfile.filepath
+        fpath = outputfile.filepath
```

Comment:
Now that we're really requiring this not be a string I think we need to adjust the datafutures description [here](https://github.com/Parsl/parsl/blob/master/docs/userguide/futures.rst) where we say:
```
hello = echo('Hello World!', outputs=['hello1.txt'])
```

Reply:
see #1640